examples/life.rs

```diff
@@ -58,7 +58,7 @@ fn iterate(z: &mut Board, scratch: &mut Board) {
 }
 
 fn turn_on_corners(z: &mut Board) {
-    let n = z.rows();
+    let n = z.nrows();
     let m = z.cols();
```

Comment:
This `cols` call escaped your purge @bluss :eyes:

Reply:
I fixed that one and another instance I found.